overlord/configstate/configcore/refresh.go

```diff
@@ -53,6 +54,15 @@ func reportOrIgnoreInvalidManageRefreshes(tr RunTransaction, optName string) err
 }
 
 func validateRefreshSchedule(tr RunTransaction) error {
+	maxInhibitionDaysStr, err := coreCfg(tr, "refresh.max-inhibition-days")
```

Comment:
Can you add  a test in configcore/refresh_test.go, something like this should do it:

```
func (s *refreshSuite) TestConfigureRefreshMaxInhibitionDays(c *C) {
	for _ , tc := []range struct{
		val interface{}
		err string
	}{
		{ val: "zzz", err: `max-inhibition-days must be a number between 1 and 14, not "zzz"` },
		{ val: "-1", err: `max-inhibition-days must be a number between 1 and 14, not "-1"` },
		{ val: "-1", err: `max-inhibition-days must be a number between 1 and 14, not "-1"` },
		{ val: "", err: `max-inhibition-days must be a number between 1 and 14, not ""` },
		{ val: "21", err: `max-inhibition-days must be a number between 1 and 14, not ""` },
		{val: 0, err: `max-inhibition-days must be a number between 1 and 14, not "0"`},
		// happy cases
		{val: 1 },
		{val: "1" },
		{val: 14 },
	} {
		err := configcore.Run(classicDev, &mockConf{
			state: s.state,
			conf: map[string]interface{}{
				"refresh.max-inhibition-days": tc.val,
			},
		})
		if tc.err != "" {
			c.Check(err, ErrorMatches, tc.err)
		} else {
			c.Check(err, IsNil)
		}
	}
}
```

Reply:
Done. I had to change some of the tests because "" and nil are "valid", in the sense that both return the same than an unconfigured entry.